### PR TITLE
feat: retrieve ConfigurationService from ConfigurationServiceProvider

### PR DESCRIPTION
**Pull request by metacosm**
Depends on #1013

### CODE REVIEW

File: operator-framework-core/src/main/java/io/javaoperatorsdk/operator/processing/Controller.java [hunk 1]
```diff
@@ -228,20 +225,7 @@ public void start() throws OperatorException {
   }
 
   private ConfigurationService configurationService() {
```
Comment (csviri) [inline operator-framework-core/src/main/java/io/javaoperatorsdk/operator/processing/Controller.java:227]: Do we need this method? Would just delete it.

**Comment by csviri** [comment]
Does this help with quarkus extension too? or the point is just to have a global config serivce?

**Review by csviri — APPROVED** [review]
LGTM , just 1 minor comment

**Comment by metacosm** [comment]
> Does this help with quarkus extension too? or the point is just to have a global config serivce?

I'm not sure yet… It started with the idea that it would help but I need to try it out…

File: operator-framework-core/src/main/java/io/javaoperatorsdk/operator/processing/Controller.java (hunk 1 above)
Comment (metacosm) [reply operator-framework-core/src/main/java/io/javaoperatorsdk/operator/processing/Controller.java]: Done.

**Pull request merged**